modules/consentManagement.js

```diff
@@ -84,6 +94,7 @@ function lookupIabConsent({onSuccess, onError}) {
         processCmpData(tcfData, {onSuccess, onError});
       } else {
         provisionalConsent = tcfData;
+        if (!iabCmpHasLoaded && timer != null) setActionTimeout();
```

Comment:
the check here should be on whether `actionTimeout` was set; without it I thik this would fire a timeout of 0.